html/mycakeapp/src/Controller/UsersController.php

```diff
@@ -47,45 +43,40 @@ public function initialize()
     }
 
     // ログイン処理
-    function login()
-    {
+    function login(){
         // POST時の処理
-        if ($this->request->isPost()) {
+        if($this->request->isPost()) {
             $user = $this->Auth->identify();
             // Authのidentifyをユーザーに設定
-            if (!empty($user)) {
+            if(!empty($user)){
                 $this->Auth->setUser($user);
-                // return $this->redirect($this->Auth->redirectUrl());
-                return $this->redirect(['controller' => 'Auction', 'action' => 'index']);
+                return $this->redirect($this->Auth->redirectUrl());
             }
             $this->Flash->error('ユーザー名かパスワードが間違っています。');
         }
     }
 
     // ログアウト処理
-    public function logout()
-    {
+    public function logout() {
         // セッションを破棄
         $this->request->session()->destroy();
         return $this->redirect($this->Auth->logout());
     }
 
     // 認証を使わないページの設定
-    public function beforeFilter(Event $event)
-    {
+    public function beforeFilter(Event $event) {
         parent::beforeFilter($event);
-        $this->Auth->allow(['login', 'index', 'add']);
+        $this->Auth->allow(['login']);
```

Comment:
```suggestion
        $this->Auth->allow(['login', 'index', 'add']);
```
ここ index と add を消さないでください。ユーザが作れないので。

Reply:
ac833b2
こちらのコミットで対応しました。